### PR TITLE
Send OS version with Downloads / Crash pixels

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/1174433894299346/1201879515350133

### Description
When triaging error reports in the Android app it would be helpful to be able to filter by OS version. This PR adds the OS pixel to the agreed pixels

### Steps to test this PR

_Download Pixels_
- [ ] Download a file using the Browser (thinkbroadband.com/download)
- [ ] Filter by Pixel Sent
- [ ] `m_download_request_started `with params: {os_version=30} {}
- [ ] `m_download_request_succeeded `with params: {os_version=30} {}

### CODE REVIEW

**Review by anikiki — APPROVED** [review]
Looks good and works as expected! :tada: There are some checks failing, didn't look at them.

**Comment by malmstein** [comment]
Thanks @anikiki !

**Review by aitorvs — CHANGES_REQUESTED** [review]
I think it'd be better if we do it through an interceptor. No class need to be modified and it would be easier to know what pixels contain the OS version

File: app/src/main/java/com/duckduckgo/app/di/StatisticsModule.kt [hunk 1]
```diff
@@ -97,7 +97,7 @@ class StatisticsModule {
         uncaughtExceptionRepository: UncaughtExceptionRepository,
         pixelSender: PixelSender,
         offlinePixels: DaggerSet<OfflinePixel>
-    ): OfflinePixelSender = OfflinePixelSender(offlinePixelCountDataStore, uncaughtExceptionRepository, pixelSender, offlinePixels)
+    ): OfflinePixelSender = OfflinePixelSender(offlinePixelCountDataStore, uncaughtExceptionRepository, pixelSender, appBuildConfig, offlinePixels)
```
Comment (anikiki) [inline app/src/main/java/com/duckduckgo/app/di/StatisticsModule.kt:100]: `appBuildConfig` is not needed anymore.

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt [hunk 2]
```diff
@@ -2552,7 +2552,10 @@ class BrowserTabViewModel(
         command.postValue(LoadExtractedUrl(extractedUrl = initialUrl))
     }
 
-    override fun onUrlExtracted(initialUrl: String, extractedUrl: String?) {
+    override fun onUrlExtracted(
+        initialUrl: String,
+        extractedUrl: String?
+    ) {
```
Comment (aitorvs) [inline app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt:2558]: nit: revert this change, this comfortably fits in the 150 char limit we have and there's no need to modify this file

File: app/src/main/java/com/duckduckgo/app/di/StatisticsModule.kt (hunk 1 above)
Comment (aitorvs) [reply app/src/main/java/com/duckduckgo/app/di/StatisticsModule.kt:100]: +1

**Review by aitorvs — APPROVED** [review]
lgtm. Left one nit about unnecessary reformat of a file we don't need to touch.

Also a +1 to a comment from @anikiki
